config.py.dist

```diff
@@ -33,6 +33,9 @@ SEGMENTS = [
 # Show the current git branch and status
     'git',
 
+# Show the number of git stashes, if any
+    'git_stash',
```

Comment:
I am hesitant to add this as a default segment for two reasons:

* Adds more processing, thus slower prompt
* I am worried about too many default segments being confusing or overloading the user.